piqueserver/config/simple.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
what for? did you mean to put this in tests perhaps?

Reply:
Yep it's for the tests. Agreed - using stringio would be neater for this.